darwin/utils.py

```diff
@@ -255,8 +253,13 @@ def parse_darwin_annotation(annotation: dict):
         main_annotation = dt.make_line(name, annotation["line"]["path"])
     elif "keypoint" in annotation:
         main_annotation = dt.make_keypoint(name, annotation["keypoint"]["x"], annotation["keypoint"]["y"])
-    elif "skeleton" in annotation:
-        main_annotation = dt.make_skeleton(name, annotation["skeleton"]["nodes"])
+    elif "ellipse" in annotation:
+        main_annotation = dt.make_ellipse(name, annotation["ellipse"])
+    elif "cuboid" in annotation:
+        main_annotation = dt.make_cuboid(name, annotation["cuboid"])
+    # TODO
+    # elif "skeleton" in annotation:
+    #     main_annotation = dt.make_skeleton(name, annotation["skeleton"]["nodes"])
```

Comment:
Fair enough, I guess we need to wait for the annotation class metadata to be included within exports

Reply:
There was also an issue when importing skeletons: fetching the annotation type id in `RemoteDataset` returns an error because it cannot find the `skeleton` type.